packages/@webex/plugin-meetings/test/unit/spec/reconnection-manager/index.js

```diff
@@ -316,29 +416,41 @@ describe('plugin-meetings', () => {
 
     describe('waitForIceReconnect()', () => {
```

Comment:
the changes inside this `describe` are not strictly related to my stuff, but I've noticed that these tests were causing the process to not exit correctly at the end of the tests, because the timer was still running for another 10s after the tests finished, so I've fixed it by using fake timers

Reply:
Thanks for fixing that